src/NuGet.Core/NuGet.Protocol/Utility/CachingUtility.cs

```diff
@@ -23,13 +24,18 @@ public static string ComputeHash(string value, bool addIdentifiableCharacters =
         {
             var trailing = value.Length > 32 ? value.Substring(value.Length - 32) : value;
             byte[] hash;
-            using (var sha = SHA1.Create())
+            using (var sha = SHA256.Create())
```

Comment:
Currently, this `ComputeHash(...)` method is used by both HTTP and plugin caching (CC @nkolev92).

Changing the hash algorithm will invalidate caching.

For HTTP caching, I believe the cache is invalidated by default after 30 minutes, but I think cache entries with SHA-1-based paths will be never be cleaned up automatically, meaning the size of the HTTP cache could theoretically double with this change. 

For plugin caching, I have not checked the behavior, but the current plugin cache entries will be invalid.

Please evaluate the impact of both changes.

Reply:
Discussed offline.

Reply:
The HTTP caching and plugin caching could be cleaned by users. So surfacing a warning to the user might not be worth the investment.